phoenix-core/src/it/java/org/apache/phoenix/end2end/ViewTTLIT.java

```diff
@@ -0,0 +1,111 @@
+package org.apache.phoenix.end2end;
+
+import com.google.common.collect.Lists;
+import org.apache.hadoop.hbase.HConstants;
+import org.apache.hadoop.hbase.client.Result;
+import org.apache.hadoop.hbase.client.ResultScanner;
+import org.apache.hadoop.hbase.client.Scan;
+import org.apache.hadoop.hbase.client.Table;
+import org.apache.hadoop.hbase.filter.*;
```

Comment:
Avoid wildcard imports. See https://phoenix.apache.org/contributing.html for Contribution guidelines. You can add preference file to your IDE.